modules/ROOT/pages/prereq-network.adoc

```diff
@@ -39,7 +39,10 @@ The version of Kubernetes supported by Anypoint Platform Private Cloud Edition u
 
 == NAT Traffic Requirements
 
-In some situation, the Kubernetes overlay network uses NAT. NAT requires that servers be able to send and receive packages with a source and destination that is different from server’s internal IP.
+In some situation, the Kubernetes overlay network uses NAT. NAT requires that servers be able to send and 
```

Comment:
situation**s**
Line 43 ...different from the server's internal IP.

Reply:
Good catch - thank you!